src/image-question/components/app.tsx

```diff
@@ -49,7 +49,12 @@ const baseAuthoringProps = deepmerge(drawingToolBaseAuthoringProps, {
   arrayMerge: (destinationArray, sourceArray) => sourceArray
 });
 
-const isAnswered = (interactiveState: IInteractiveState | null) => !!interactiveState?.answerText;
+export const isAnswered = (interactiveState: IInteractiveState | null, authoredState?: IAuthoredState) => {
+  const hasAnswerPrompt = !!authoredState?.answerPrompt;
+  const hasAnswerText = !!interactiveState?.answerText;
+  // image questions without answer prompts are always answered otherwise they are answered when there is answer text present
+  return !hasAnswerPrompt || hasAnswerText;
```

Comment:
They should require some kind of work in the drawing area. This could be actual drawing, or it could just be an snapshot or uploaded image.  Perhaps just having `interactiveState` is enough to determine this.

Reply:
So you want me to change the answer check altogether? Previously it was just checking the answerText.